1.8/administration/logging/elk.md

```diff
@@ -1,147 +1,200 @@
 ---
-post_title: Log Management with ELK
-nav_title: ELK
+post_title: Log Management in DC/OS with ELK
 menu_order: 1
+enterprise: 'yes'
 ---
-You can pipe system and application logs from a DC/OS cluster to your existing ElasticSearch, Logstash, and Kibana (ELK) server.
+
+You can pipe system and application logs from the nodes in a DC/OS cluster to your existing ElasticSearch, Logstash, and Kibana (ELK) server. This article recommends to store all unfiltered logs directly on ElasticSearch, and perform filtering/specialized querying on ElasticSearch directly. Consequently, the suggested pipeline sends the output from filebeat in the nodes directly to a centralized ElasticSearch instance, without using Logstash. If you're interested in using Logstash for log processing or parsing, consult the [Filebeat][2] and [Logstash][8] documentation.
+
+NOTE: This article does NOT cover how to set up secure TLS communication between the Filebeat instances and ElasticSearch. For details on how to achieved this, please check the [Filebeat][2] and [ElasticSearch][5] documentation.
 
 **Prerequisites**
 
-These instructions are based on CoreOS 766.4.0 and might differ substantially from other Linux distributions. This document does not explain how to setup and configure an ELK server.
+These instructions are based on CentOS 7 and might differ substantially from other Linux distributions.
 
-*   A recent ELK stack with an HTTPS interface and certificate.
-*   All DC/OS nodes must be able to connect to your ELK server via HTTPS.
-*   The `ulimit` of open files must be set to `unlimited` for your user with root access.
+*   All DC/OS nodes must be able to connect to your ElasticSearch server on the port used for ElasticSearch/Filebeat communication (9200 by default)
 
 # <a name="all"></a>Step 1: All Nodes
 
 For all nodes in your DC/OS cluster:
 
-1.  Install Elastic's [logstash-forwarder][2]. The project is written in Go and is compiled for most platforms.
-2.  Download the HTTPS certificate for your ELK stash. In our examples, we assume the certificate is named `elk.crt`.
+1.  Install Elastic's [filebeat][2]. Installers are available for most major platforms.
+
+```bash
+curl -L -O https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-5.0.0-x86_64.rpm
```

Comment:
Global: Add `$` before commands. This is our established convention throughout the docs.